### PR TITLE
Add common open containers LABELS

### PR DESCRIPTION
**Pull request by jason-fox**
-  Add common opencontainers LABELS
-  ~Alter build/post-push hooks to include LD build~
-  Remove MAINTAINER as deprecated.

### CODE REVIEW

**Comment by jason-fox** [comment]
A manual LD build is now possible, but no automated LD image is being build. This is a minimal change to build both NGSI and NGSI-LD images.

File: docker/cygnus-ngsi-ld/Dockerfile [hunk 1]
```diff
@@ -346,3 +346,13 @@ ENTRYPOINT ["/cygnus-entrypoint.sh"]
 # Ports used by cygnus-ngsi
 EXPOSE ${CYGNUS_SERVICE_PORT} ${CYGNUS_API_PORT}
 
+LABEL "maintainer"="Cygnus Team. Telefónica I+D"
+LABEL "org.opencontainers.image.authors"="iot_support@tid.es"
+LABEL "org.opencontainers.image.documentation"="https://fiware-cygnus.rtfd.io/"
+LABEL "org.opencontainers.image.vendor"="Telefónica Investigación y Desarrollo, S.A.U"
+LABEL "org.opencontainers.image.licenses"="AGPL-3.0-only"
+LABEL "org.opencontainers.image.title"="Cygnus-LD Data Peristence"
```
Comment (fgalan) [inline docker/cygnus-ngsi-ld/Dockerfile:354]:
```suggestion
LABEL "org.opencontainers.image.title"="Cygnus-LD Data Persistence"
```

Similar typo in other LABEL in this PR

File: docker/cygnus-ngsi/Dockerfile [hunk 2]
```diff
@@ -351,3 +351,12 @@ ENTRYPOINT ["/cygnus-entrypoint.sh"]
 # Ports used by cygnus-ngsi
 EXPOSE ${CYGNUS_SERVICE_PORT} ${CYGNUS_API_PORT}
 
+LABEL "maintainer"="Cygnus Team. Telefónica I+D"
+LABEL "org.opencontainers.image.authors"="iot_support@tid.es"
+LABEL "org.opencontainers.image.documentation"="https://fiware-cygnus.rtfd.io/"
+LABEL "org.opencontainers.image.vendor"="Telefónica Investigación y Desarrollo, S.A.U"
+LABEL "org.opencontainers.image.licenses"="AGPL-3.0-only"
+LABEL "org.opencontainers.image.title"="Cygnus Data Peristence"
```
Comment (AlvaroVega) [inline docker/cygnus-ngsi/Dockerfile:359]: Peristence  -> Persistence

Comment (jason-fox) [reply docker/cygnus-ngsi/Dockerfile:359]: Typo Fixed 2fbc66f

File: docker/cygnus-ngsi-ld/Dockerfile (hunk 1 above)
Comment (jason-fox) [reply docker/cygnus-ngsi-ld/Dockerfile]: Typo Fixed 34b9ea2

**Comment by IvanHdzC** [comment]
Tecnically, this PR looks good, but implementation doesn't fit project conventions.

Take cygnus-twitter example. It has a very similar case than cygnus-ngsi-ld. It is an entire new artifact wich is created as a new image on [dockerhub](https://hub.docker.com/repository/docker/telefonicaiot/fiware-cygnus-twitter). This is because it has it's own build configs and triggers on docker hub. 

If we accept this PR, every image created for cygnus-ngsi artifact will trigger a new cygnus-ngsi-ld image, and that doesn't seem right, since both have different source codes.

We might create a new repository on the "telefonicaiot" group for this image soon.

**Comment by jason-fox** [comment]
@IvanHdzC  - I agree `docker/cygnus-ngsi/hooks/build`  and `docker/cygnus-ngsi/hooks/post-push` may need moving elsewhere, but since I have no visibility of your Docker Hub infrastructure I can't work out where. The point is that the Cygnus-LD needs building **somehow**  🚧  👷 🚧 

Maybe just replace this change on your own PR and close this? I'll leave the branch up for now.

File: docker/cygnus-ngsi/hooks/build [hunk 3]
```diff
@@ -17,7 +17,18 @@
...
+	-t $IMAGE_NAME \
+	.
+
+docker image build --build-arg SOURCE_BRANCH=$SOURCE_BRANCH \
+	--label "org.opencontainers.image.created"=$(date +%Y-%m-%d) \
+	--label "org.opencontainers.image.revision"=$SOURCE_COMMIT \
+	--label "org.opencontainers.image.version"=$DOCKER_TAG \
+	-f docker/cygnus-ngsi-ld/Dockerfile \
+	-t $IMAGE_NAME-ld \
+	.
```
Comment (IvanHdzC) [inline docker/cygnus-ngsi/hooks/build:34]:
```suggestion
```

**Comment by IvanHdzC** [comment]
> @IvanHdzC - I agree `docker/cygnus-ngsi/hooks/build` and `docker/cygnus-ngsi/hooks/post-push` may need moving elsewhere, but since I have no visibility of your Docker Hub infrastructure I can't work out where. The point is that the Cygnus-LD needs building **somehow** construction construction_worker construction
> 
> Maybe just replace this change on your own PR and close this? I'll leave the branch up for now.

Ok, @jason-fox could you please remove both post-build and new build from your branch/PR so we can accept and merge the labels changes please?

After that we can build the new image.

File: docker/cygnus-ngsi/hooks/build (hunk 3 above)
Comment (jason-fox) [reply docker/cygnus-ngsi/hooks/build]: Fixed ef6156a

**Comment by jason-fox** [comment]
> Ok, @jason-fox could you please remove both post-build and new build from your branch/PR so we can accept and merge the labels changes please?

Fixed ef6156a

**Review by IvanHdzC — APPROVED** [review]
LGTM

**Review by fgalan — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**